moped-database/migrations/1654646507541_proj_list_view_fund_src_filter/up.sql

```diff
@@ -0,0 +1,97 @@
+CREATE OR REPLACE VIEW "public"."project_list_view" AS 
+ WITH project_person_list_lookup AS (
+         SELECT mpp.project_id,
+            string_agg(DISTINCT concat(mu.first_name, ' ', mu.last_name, ':', mpr.project_role_name), ','::text) AS project_team_members
+           FROM ((moped_proj_personnel mpp
+             JOIN moped_users mu ON ((mpp.user_id = mu.user_id)))
+             JOIN moped_project_roles mpr ON ((mpp.role_id = mpr.project_role_id)))
+          WHERE (mpp.status_id = 1)
+          GROUP BY mpp.project_id
+        ), funding_sources_lookup AS (
+         SELECT mpf_1.project_id,
+            string_agg(mfs.funding_source_name, ', '::text) AS funding_source_name
+           FROM (moped_proj_funding mpf_1
+             LEFT JOIN moped_fund_sources mfs ON ((mpf_1.funding_source_id = mfs.funding_source_id)))
+          WHERE (mpf_1.funding_status_id != 0)
```

Comment:
I updated this line from `WHERE (mpf_1.funding_status_id = 1)` to `WHERE (mpf_1.funding_status_id != 0)` since this column currently represents both soft delete status (0) and funding statuses (1 = Tentative ,2 = Confirmed, etc.)